editor/scene_tree_dock.cpp

```diff
@@ -2557,6 +2557,11 @@ void SceneTreeDock::_focus_node() {
 }
 
 void SceneTreeDock::attach_script_to_selected(bool p_extend) {
+	if (ScriptServer::get_language_count() == 0) {
+		EditorNode::get_singleton()->show_warning(TTR("Cannot attach a script: there are no languages registered."));
```

Comment:
Just in case someone wonders why this message is shown:

```suggestion
		EditorNode::get_singleton()->show_warning(TTR("Cannot attach a script: there are no languages registered.\nThis is probably because this editor was built with all language modules disabled."));
```

Reply:
Just wondering too: I'm not sure whether `TTR` strings can be broken down into several strings for readability, cannot they?

Reply:
Not yet, but this should be feasible once the extraction script supports C++11 raw literals (`R"(text here)"`).

Reply:
Yeah, it still exceeds the 120 chars line, especially indenting with two tabs, so I don't want to bother...